duckduckgo/Android
# Feature/omnibar improvements

Asana:
https://app.asana.com/0/471012189846020/477327275839578

**Description**:


**Steps to test this PR**:
Generally, just give the tires a good kicking and make sure it holds up.
1. Compare omnibar looks to design, noting the excluded parts of the task as per the description.
1. Run some searches, see how it handles
1. Ensure the URL is updated when navigating around
1. Ensure the progress bar shows and updates as pages are loading
1. Ensure the toolbar scrolls away when you scroll down a web page
1. Ensure the toolbar returns when you scroll back up, or whenever you navigate between pages 
1. Rotate device, edit url, tap clear button etc...




###### Author Checklist:
- [x] Ensure that there is a testing strategy (and document any non-automated tests)
- [x] Consider implications (CPU, memory, database / preference migrations, statistics)


###### Reviewer Checklist:
